### ISSUE TITLE
[Browser Bookmarks] Can't search Chrome bookmarks

### ISSUE THREAD

**Issue by Janzee** [raycast/extensions#15895]
### Extension

https://raycast.com/raycast/browser-bookmarks

### Raycast Version

1.87.5

### macOS Version

14.6.1

#### Error:
```
Error: EPERM: operation not permitted, open '~/Library/Application Support/Google/Chrome/Local State'
```

### Steps To Reproduce

1. search anything about brower bookmarks
2. an error occured says operation not permitted

**Comment by selfish** [comment]
Seems like an access issue, not a bug with Raycast.
Is this directory accessible to you on finder?

**Comment by selfish** [comment]
Also, seems related to #15895

**Comment by Janzee** [comment]
> Seems like an access issue, not a bug with Raycast. Is this directory accessible to you on finder?

Yes, this directory is accessible on finder. Also, I have tried these, which all failed:
1. execute chmod a+r for this directory in Terminal
2. add Raycast to 'Full Disk Access' list in system settings

So, any idea?

**Comment by thomaslombart** [comment]
@zeghaun @fanace @jarviszhb let's continue the discussion here to avoid duplicates and different messages scattered on different issues.

Could one of you let me know if this issue also happens on Chromium browsers like Brave or Microsoft Edge?

**Comment by AOYLAOTANG** [comment]
@thomaslombart I tried edge and it occured a new tips 'Cannot read properties of undefined (reading 'path')'
![image](https://github.com/user-attachments/assets/af35f385-88e5-4e73-a1c5-28dee8133ec5)

**Comment by yantze** [comment]
This is the situation I encountered

![Image](https://github.com/user-attachments/assets/0f0b19ac-967b-4a5b-a7fd-657df2166e71)

and terminal show:

![Image](https://github.com/user-attachments/assets/d6d70924-8cd7-47d9-a5b6-185f537349ac)

**Comment by Ezreal09** [comment]
I have the same problem

**Comment by reallinzc** [comment]
alibaba？

**Comment by alvincross** [comment]
I have met the same problem

**Comment by jarviszhb** [comment]
@thomaslombart I found the cause of this bug. When only Chrome is installed on my Mac, the plugin runs normally. However, when both Chrome and Edge are installed, the bug is happened.

**Comment by thomaslombart** [comment]
Interesting! Thanks for looking into it @jarviszhb.

@selfish @alvincross @reallinzc @Ezreal09 @yantze @AOYLAOTANG is that the same for you?

**Comment by lessonli** [comment]
i hage met the same problem

**Comment by lessonli** [comment]
> alibaba？

我家里的个人电脑也有同样的问题

**Comment by lessonli** [comment]
> [@thomaslombart](https://github.com/thomaslombart) I found the cause of this bug. When only Chrome is installed on my Mac, the plugin runs normally. However, when both Chrome and Edge are installed, the bug is happened.

我 卸载了 edge 问题并没有解决

**Comment by ObedWang** [comment]
i have met the same problem

**Comment by yantze** [comment]
> Interesting! Thanks for looking into it [@jarviszhb](https://github.com/jarviszhb).
> 
> [@selfish](https://github.com/selfish) [@alvincross](https://github.com/alvincross) [@reallinzc](https://github.com/reallinzc) [@Ezreal09](https://github.com/Ezreal09) [@yantze](https://github.com/yantze) [@AOYLAOTANG](https://github.com/AOYLAOTANG) is that the same for you?

Previously installed, but later uninstalled.

**Comment by zeghaun** [comment]
chrome : 138.0.7204.93
raycast：1.100.3


I try  alfred , utools。 There is a similar issue。

I agree with the previous response, it could be due to Chrome's security policy preventing other software from reading bookmark information.

![Image](https://github.com/user-attachments/assets/31fe817f-d407-4f22-b5ce-2f7d5c9f9bad)

**Comment by zeghaun** [comment]
@thomaslombart @raycastbot

**Comment by waitsilently** [comment]
excute "open /Users/username/Library/Application Support/Google/Chrome/Local State" failed,
excute "open /Users/username/Library/Application\ Support/Google/Chrome/Local\ State" will success.
I guess add "/" before space can solve this problem.

**Comment by lessonli** [comment]
@selfish 我认为@waitsilently 说的很有道理 是不是可以尝试修复一版本

**Comment by waitsilently** [comment]
您好，邮件已收到。祝您心情愉快。

**Comment by wardchan** [comment]
Hi guys. This issue is likely related to `Google Chrome` using `Default` as the profile ID by default. It seems some machines use `Profile [number]` as the profile ID. I created the following PR to try to fix it. 🙏

https://github.com/raycast/extensions/pull/21706

**Comment by waitsilently** [comment]
您好，邮件已收到。祝您心情愉快。

**Comment by waitsilently** [comment]
您好，邮件已收到。祝您心情愉快。

**Comment by waitsilently** [comment]
您好，邮件已收到。祝您心情愉快。
